packages/compartment-mapper/test/fixtures-cjs-compat/node_modules/app/index.js

```diff
@@ -47,9 +47,7 @@ module.exports.assertions = {
     try {
       callself.makeTheCall();
     } catch (e) {
-      throw Error('Referencing exported functions should be possible', {
-        cause: e,
-      });
+      throw Error('Referencing exported functions should be possible. \n' + e.stack);
```

Comment:
We should not inline the error stack of the cause in the message.

```suggestion
      throw Error(`Referencing exported functions should be possible, because ${cause.message}`, {cause});
```

Also, prefer `cause` or `error` to `e` for the catch clause.

Reply:
Node 16.something introduced support for passing cause to error, but it didn't propagate for me (might be ava or some other reason it's dropped) I needed it for debugging.